tutorials/R/Geospatial-skills/intro-vector-r/00-open-a-shapefile/00-open-a-shapefile.Rmd

```diff
@@ -85,7 +85,7 @@ with: point, line or polygon.
 
 <figure>
     <a href="https://raw.githubusercontent.com/NEONScience/NEON-Data-Skills/dev-aten/graphics/dc-spatial-vector/pnt_line_poly.png">
-    <img src="https://raw.githubusercontent.com/NEONScience/NEON-Data-Skills/dev-aten/graphics/dc-spatial-vector/pnt_line_poly.png"></a>
+    <img src="https://raw.githubusercontent.com/NEONScience/NEON-Data-Skills/dev-aten/graphics/dc-spatial-vector/pnt_line_poly.png"> alt = "A figure detailing the three different types of vector objects: points, lines, and polygons." </a>
```

Comment:
Two notes here:

1.  the alt="..." must be within the closing bracket of the <img> tag (i.e., move the ">" to after the alt="..." argument)
2. I didn't know this until this afternoon, but alt text is best when it doesn't begin with "A figure.." or "A diagram..". I just added a new link to the section on Alt text (https://www2.le.ac.uk/webcentre/plone/build/basics/add-images/alt-text). Perhaps you could just do: "The three different types of vector objects: points, lines, and polygons"

Suggest:
`<img src="https://raw.githubusercontent.com/NEONScience/NEON-Data-Skills/dev-aten/graphics/dc-spatial-vector/pnt_line_poly.png" alt = "A figure detailing the three different types of vector objects: points, lines, and polygons."> 
</a> `
with the closing tag `</a>` on a new line.